scaffolder-templates/basic-workflow/skeleton/README.md

```diff
@@ -41,12 +41,12 @@ curl -v -X POST -H "Content-Type: application/json" http://localhost:8080/${{ va
 # Dev UI Tools
 Use the `Serverless Workflow Tools` to manage and monitor the published workflows:
 ```bash
-open -t http://localhost:8080/q/dev/org.kie.kogito.kogito-quarkus-serverless-workflow-devui/workflowInstances
+open -t http://localhost:8080/q/dev-v1/org.kie.kogito.kogito-quarkus-serverless-workflow-devui/workflowInstances
```

Comment:
btw, open is unrecognized in fedora/rhel

Reply:
Removed and kept the URLs only.